README.md

```diff
@@ -102,6 +102,23 @@ After calling `start_nodes/2`, you will receive a list of node names you can the
 to communicate with via RPC or however you'd like. Although they're automatically cleaned
 up when the calling process dies, you can manually stop nodes as well to test disconnection.
 
+`start_nodes/3` accepts list of options as an additional parameter.
+Two options are currently supported: `:app_names` and `:files`, as described below.
+
+Use option `:app_names` with an ordered list of application names for scenarios
```

Comment:
I would prefer `:applications`, to be in line with other places such a list is used.

Reply:
I think I also prefer `:include` rather than `:files`, thoughts?